src/controllers/user.ts

```diff
@@ -0,0 +1,105 @@
+import type { Request, Response } from 'express';
+import { v4 as uuidv4 } from 'uuid';
+
+import type { SuggestRequestQueryModel, UserModel } from '../models/User';
+
+const db: { users: UserModel[] } = {
+  users: [
+    {
+      id: '1',
+      password: '1u23',
+      age: 5,
+      login: 'Sasha',
+      isDeleted: false,
+    },
+    {
+      id: '2',
+      password: '1e23',
+      age: 25,
+      login: 'Senya',
+      isDeleted: false,
+    },
+    {
+      id: '3',
+      password: '1c23',
+      age: 35,
+      login: 'Sonya',
+      isDeleted: false,
+    },
+    {
+      id: '4',
+      password: '12b3',
+      age: 15,
+      login: 'Sergey',
+      isDeleted: false,
+    },
+    {
+      id: '4',
+      password: '123a',
+      age: 25,
+      login: 'Sveta',
+      isDeleted: false,
+    },
+  ],
+};
+
+const create = (req: Request, res: Response) => {
+  const { login, age, password }: UserModel = req.body;
+
+  if (!(login && password && age)) {
```

Comment:
Request parameters validation should be a separate middleware. Use libraries like joi for a more convenient validation